k9mail-library/src/main/java/com/fsck/k9/mail/store/webdav/WebDavStore.java

```diff
@@ -28,17 +35,24 @@
 import javax.xml.parsers.SAXParser;
 import javax.xml.parsers.SAXParserFactory;
 
-import java.io.*;
+import java.io.BufferedReader;
```

Comment:
Our [code style settings](https://github.com/k9mail/k-9/wiki/CodeStyle) should place the `java…` imports at the top.